config/api.yml

```diff
@@ -117,6 +117,17 @@
     :klass: ChargebackRate
     :subcollections:
     - :rates
+  :container_deployments:
+    :description: Container Provider Deployment
+    :identifier: container_deployment
+    :klass: ContainerDeployment
+    :methods: *70174834084700
+    :options:
+    - :collection
```

Comment:
@alongoldboim Can you respond about this?  /cc @simon3z 
seems that this is blocking @imtayadeway from giving a 👍